### PR TITLE
Update Fileshare-to-ODSP-migration-guide.md

### PR DESCRIPTION
**Pull request by panjkov**
Updated to correct link to Microsoft 365 adoption guide in section "Onboarding related resources"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/Fileshare-to-ODSP-migration-guide.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Develop a plan to prepare your users for the upcoming change. Consideration fact
 - Let them know whether individuals can opt out of the migration process.
 
 ### Onboarding related resources
-- [Microsoft 365 adoption guide](https://devfasttrackv4storage.blob.core.windows.net/marketing/en-us/resources/Microsoft%20365%20User%20Adoption%20Guide.pdf): Outlining methodology and resources for implementing proven adoption success factors
+- [Microsoft 365 adoption guide](https://docs.microsoft.com/en-us/office365/customlearning/champ_o365guide): Outlining methodology and resources for implementing proven adoption success factors
```
Comment (JohanFreelancer9) [inline migration/Fileshare-to-ODSP-migration-guide.md:178]:
```suggestion
- [Microsoft 365 end user adoption guide](https://docs.microsoft.com/office365/customlearning/champ_o365guide): Outlining methodology and resources for implementing proven adoption success factors
```

**Comment by panjkov** [comment]
@scanum @JohanFreelancer9 I just changed the URL and did not change any text in the document. I'm not 100% sure but I believe that copy editing is not necessary.

File: migration/Fileshare-to-ODSP-migration-guide.md (hunk 1 above)
Comment (JohanFreelancer9) [reply migration/Fileshare-to-ODSP-migration-guide.md:178]: Please remove the locale from the link.

**Comment by JohanFreelancer9** [comment]
@panjkov Thank you, looks good. We do not include locales (e.g. en-us) in the links, and also use the full title of the referenced article where possible.

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**